.github/workflows/ci-cd.yml

```diff
@@ -1,7 +1,7 @@
 ---
 name: "CI/CD: Continuous integration and continuous deployment"
 
-"on":
```

Comment:
Oops, my mistake that this was changed in mqtt-pulsar-forwarder. yamllint, part of super-linter, will complain if we remove the double quotes. (I was surprised that this truthy problem extends to keys in YAML: https://github.com/adrienverge/yamllint/issues/158 )

Reply:
Actually, I do not know if GitHub changes the default configuration of yamllint to accept `on:` instead of `"on":`. The other workflow files would require fixes as well, so let's leave yamllint for later.